### PR TITLE
Add support to Laravel 6.0

### PR DESCRIPTION
**Pull request by FrancaR**

### CODE REVIEW

File: composer.json
Comment (owenvoke) [inline composer.json]: This may be better as `^5.7 || ^6.0` as using an unbound version constraint is [less secure](https://getcomposer.org/doc/faqs/why-are-unbound-version-constraints-a-bad-idea.md)?

File: composer.json
Comment (owenvoke) [inline composer.json]: I guess Testbench needs to be updated to support Laravel 6.0 as well (Testbench `~3.0 || ^4.0`)?

~However, version 4 isn't released yet.~

**Review by owenvoke — COMMENTED** [review]
Just a couple of minor queries about the versions specified in Composer. 🙂

File: composer.json
Comment (FrancaR) [reply composer.json]: Good note! Updated!

File: composer.json
Comment (FrancaR) [reply composer.json]: Good note! Updated!

**Review by owenvoke — APPROVED** [review]
LGTM!

File: composer.json
Comment (Elhebert) [inline composer.json]: I think `"^6.0"` would be better.

Then I just tag it as a "major" release that introduces breaking changes.

File: composer.json
Comment (Elhebert) [inline composer.json]: Same here.

As `4.0` is only required from Laravel 6.0, it makes more sense to only require it and tag it as a major release.

**Review by Elhebert — CHANGES_REQUESTED** [review]
I wouldn't make this PR backward compatible.

I'll tag the new release as a new major one that introduces breaking changes. That way projects that use Laravel up to 5.8 don't have to update anything.

Other than that, thx for your effort!

File: composer.json
Comment (owenvoke) [reply composer.json]: Just a note, if the package is going to become Laravel 6.0 and above, I guess PHP needs to be updated to require `^7.2`, Travis needs to be updated to not run against `7.1`, and PHPUnit needs to be updated to `^8.0` (otherwise [this happens](https://travis-ci.com/pxgamer/laravel-sri/jobs/231031190)). 👍

With [these changes](https://github.com/pxgamer/laravel-sri/compare/59ff56a78fcef90163e1b2c5d73c9b19a3c55816...b5e200e8e449d05f89ca9f78e2ad39c9224e9c50), you end up with [Travis successfully passing](https://travis-ci.com/pxgamer/laravel-sri/builds/125866489) (albeit, a couple of deprecation warnings).

Comment (Elhebert) [reply composer.json]: Good point.

I've been out of the Laravel (and PHP) world for a couple of months now, so I haven't really been able to keep up with what was happening with the releases 😂

**Review by owenvoke — APPROVED** [review]

**Comment by FrancaR** [comment]
@Elhebert As you will launch a major release, i'm have more PR. Can you merge this PR and not release the new version yet?

**Comment by owenvoke** [comment]
Sounds interesting @FrancaR 👍 Do the other PRs require a major version bump? Just interested as this seems to be the only package I'm waiting on atm.

**Comment by FrancaR** [comment]
> Sounds interesting @FrancaR  Do the other PRs require a major version bump? Just interested as this seems to be the only package I'm waiting on atm.

@pxgamer because when I added the support to Laravel 6.0, I'm forgot of create in a new branch. Now, with these new PRs, I created the branch based on master (with Laravel 6.0 support).

The new PR is:
1) Add descriptions of options in config file.
2) Actually, when use Blade directives, you can be make `@assetSRI(/css/app.css)`, in my PR, you too can be make `@assetSRI("/css/app.css")` or `@assetSri('/css/app.css')`. Based on #16.

What's your think about this?

**Comment by owenvoke** [comment]
I guess the first idea could be added in a minor/patch release as this is just adding comments.

The 2nd one, I'd personally really like that as this has bugged me for a while, but I didn't get round to fixing it. If it's backwards compatible, I guess this could be a minor release after the Laravel 6 changes. :+1: But if it's a quick fix, it makes sense to include it in the major change (up to @elhebert though).

---

On another note, just noticed that Travis doesn't seem to be running. Is it not enabled on the repository? 🙂

**Review by Elhebert — CHANGES_REQUESTED** [review]
Can you also update the Travis config to also remove PHP 7.1?

**Comment by Elhebert** [comment]
For the other changes, I would do it in another PR to be able to merge this one as soon as possible and unblock people that need it.

Then we can also merge them into the "old" version as well as the new one (if the changes are compatible).

Thanks again! 👌

**Review by Elhebert — APPROVED** [review]

**Comment by Elhebert** [comment]
Thx!

Merged and released!

If I get some time I'll rework the README.md and publish a new branch for the "legacy" version.

**Pull request merged**
